lib/project_types/rails/commands/create.rb

```diff
@@ -143,7 +143,7 @@ def build(name, db)
         end
 
         CLI::UI::Frame.open(@ctx.message('rails.create.running_generator')) do
-          syscall(%w(rails generate shopify_app))
```

Comment:
Perhaps this could be a more generic flag such as `--new-app`, meaning:

* The CLI doesn't have to know about this specific detail of `shopify_app`
* If similar behaviour is required in future, we don't need to add further flags.

Reply:
I like that idea, but should we name it more explicitly like `--new-shopify-cli-app`?

Reply:
Does this mean we need to change the option in `shopify_app` to `--new-shopify-cli-app`?

Reply:
I think so, as Andy mentioned it will make it a lot easier to make similar changes in `shopify_app` in the future without having to tweak the CLI every time.

Reply:
Just put the PR up to introduce the new flag: https://github.com/Shopify/shopify_app/pull/1193

Reply:
Latest commit updates this name as well!